bin/prerenderio-bust.sh

```diff
@@ -1 +1,4 @@
+
```

Comment:
I don't know if it has to be line 1, but to be sure lets nichts on the white space on line one.